### PR TITLE
DTS-36197:Improvements on KH widgets for EJ Project

### PR DESCRIPTION
**Pull request by gipathak**
DTS-36197:Improvements on KH widgets for EJ Project

### CODE REVIEW

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_920/UpdateMeanTimeToRecover.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.publicissapient.kpidashboard.apis.mongock.rollback.release_920;
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_920/UpdateMeanTimeToRecover.java:1]: add copyright

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_920/UpdateMeanTimeToRecover.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package com.publicissapient.kpidashboard.apis.mongock.upgrade.release_920;
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_920/UpdateMeanTimeToRecover.java:1]: add copyright

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_920/UpdateMeanTimeToRecover.java (hunk 1 above)
Comment (gipathak) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_920/UpdateMeanTimeToRecover.java:17]: Done

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_920/UpdateMeanTimeToRecover.java (hunk 2 above)
Comment (gipathak) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_920/UpdateMeanTimeToRecover.java:17]: Done

**Review by shunaray — APPROVED** [review]

**Review by Chittauri — APPROVED** [review]

**Pull request merged**